### PR TITLE
Feature/game details

### PR DESCRIPTION
**Pull request by daomeow**
# Describe Changes
- Added a select game action and reducer 
- Made Dashboard aware of when a game is clicked [@ line 14]
- Added our GameDetails component to Router 
- #3 

# Type of Change
- [ ] Feature HTML
- [X] Feature JavaScript
- [ ] Feature CSS 
- [ ] Refactor/Fix Bugs

# Please describe the change made
- See above 

Please give a short description of the changes that were made below
- Fix: N/A 

- Feature: Please see above 

- Blocker: Unable to render developers/designers currently the same way on in GameDetails. Error message: "Objects are not valid as a React child". I will continue to work on this and the rest of my tasks for today, but if you have insight please let me know 🙏 

# How has this been tested?

- [X] In Browser
- [ ] In Console

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ function App() {
             path='/wishlist'
             component={Wishlist}
           />
+          <Route exact
+            path='/:id'
```
Comment (Ryan-D-Miller) [inline src/components/App/App.js:42]: Do you think we should put a route before the id like game/:id? for a project this small probably not a big deal

File: src/components/GameDetails/GameDetails.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+function GameDetails() {
+  const game = useSelector(state => state.gameDetails);
+
+  return(
+    <div className='game-details'>
+        <img src={game.thumb_url}/>
+        <h1>{game.name}</h1>
+        <p>{game.price}</p>
+        <button>Add to wish list</button>
+      {/* <p>{game.designers}{game.developers}</p> */}
```
Comment (errabun) [inline src/components/GameDetails/GameDetails.js:14]: We can vote on what information we care to display here!

**Review by errabun — APPROVED** [review]
Looks awesome!  Getting close to the end of this project!!

File: src/components/GameDetails/GameDetails.js (hunk 2 above)
Comment (daomeow) [reply src/components/GameDetails/GameDetails.js:14]: Sounds good! I was going off the wireframe

File: src/components/App/App.js (hunk 1 above)
Comment (daomeow) [reply src/components/App/App.js:42]: I was thinking if we had extra time at the end we could set up Router so if a user typed forward-slash + game name, the page would render that board game's details. We could use an effect hook to check if the name is there, grab the name, user a helper function to access all the games, find the matched name then use the action to send it to the store? 

(I wrote this out mostly as a note to self 😂 )

**Pull request merged**